app/controllers/bonus_credits_controller.rb

```diff
@@ -3,17 +3,20 @@ class BonusCreditsController < ApplicationController
   before_action :authenticate_control!
 
   def index
-    @bonus_credits = BonusCredit.where(game: current_game).order(:round, :team_name, :amount)
+    @bonus_credits = BonusCredit.where(game: current_game).order(:round, :team_id, :amount)
   end
 
   def new
     @round = current_game.round
-    @countries = Game::COUNTRIES
     @bonus_credits = BonusCredit.new
+    @teams = Team.all
```

Comment:
Having situations where you refer to Team.all is really dicey.

A Team is a collection of users tied to a single game.  Brazil Team's income in one game is different from another Brazil Team's Income in another game.  As such, please write a migration to add game_id to Team.  A Team should be created tied to a Game (ie. game_id cannot be nil).

Then you can reference a game's teams as `current_game.teams` or Game.find(1).teams.